main.py

```diff
@@ -95,29 +133,23 @@ def register_user(request: CreateUser):
         access_token = create_access_token({"sub": request.email}, expires)
 
         return {"message": "User registered successfully", "access_token": access_token}
-    except Exception as generic_exception:
-        # Handle specific errors if needed
-        error_message = str(generic_exception)
-        raise HTTPException(
-            status_code=500, detail=f"Internal server error: {error_message}"
-        ) from generic_exception
 
 
-@app.post("/auth")
+@app.post("/dev/auth")
```

Comment:
this shouldn't be hardcoded

Reply:
This is a requirement from the serverless framework intended to only work locally.
I will remove